feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64fb0fa0968f2b113b2d90e9.md

---
id: 64fb0fa0968f2b113b2d90e9
title: Step 40
challengeType: 0
dashedName: step-40
---

# --description--

As you did in the `deleteTask` function, you need to find the index of the task to be edited.

> Create a `dataArrIndex` variable. For the value, use `findIndex()` on `taskData`, pass in `item` as the parameter of its callback, then check if the `id` of `item` is equal to the `id` of the `parentElement` of `buttonEl`.

Reviewer: ```suggestion
Create a `dataArrIndex` variable. For its value, utilize the `findIndex()` method on `taskData`. Pass `item` as the parameter to its callback function and check if the `id` of `item` is equal to the `id` of the `parentElement` of `buttonEl`.
```
